app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/declaration/Declaration.scala

```diff
@@ -111,7 +111,8 @@ case class DeclarationJourney(sessionId: SessionId,
                               maybeEori: Option[Eori] = None,
                               maybeJourneyDetails: Option[JourneyDetails] = None,
                               maybeGoodsDestination: Option[GoodsDestination] = None,
-                              maybeValueWeightOfGoodsExceedsThreshold: Option[Boolean] = None)
+                              maybeValueWeightOfGoodsExceedsThreshold: Option[Boolean] = None,
+                              maybeSearchGoodsCountry: Option[String] = None)
```

Comment:
There's already a GoodsEntry with maybeCountryOfPurchase. Is this doing something different?

Reply:
Will look into it

Reply:
Thanks.

Reply:
Find well enough for this ticket.